### PR TITLE
Initial checkin of rna-collect-rna-seq-metrics workflow.

### LINKED ISSUE #16: Add collect-rna-seq-metrics workflow
For AMP PD RNA data, a collect-rna-seq-metrics workflow was run.

### PR DESCRIPTION
**Pull request by wnojopra**
This PR includes the rna-collect-rna-seq-metrics workflow which executes picard CollectRnaSeqMetrics on a BAM file.

Fixes #16 .

### CODE REVIEW

File: rna/rna-collect-rna-seq-metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# RNACollectRnaSeqMetrics with Picard
+
+This workflow runs `picard CollectRnaSeqMetrics` on a BAM file
```
Comment (mbookman) [inline rna/rna-collect-rna-seq-metrics/README.md]: This workflow runs [picard CollectRnaSeqMetrics](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.0.0/picard_analysis_CollectRnaSeqMetrics.php) on a BAM file.

File: rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.inputs.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "##_COMMENT1": "Set your per-sample values: name and BAM path, and intervals file path.",
+  "RNACollectRnaSeqMetrics.sample_name": "__INSERT_SAMPLE_NAME__",
+  "RNACollectRnaSeqMetrics.bam_file": "__INSERT_GCS_PATH_TO_BAM_FILE__",
+
+  "##_COMMENT2": "Path to ref_flat and intervals file.",
+  "RNACollectRnaSeqMetrics.ref_flat_file": "__INSERT_GCS_PATH_TO_REF_FLAT_FILE__",
```
Comment (mbookman) [inline rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.inputs.json]: gs://amp-pd-transcriptomics/inputs/reference/picard/gencode.v29.primary_assembly.annotation.Reflat_Picard.txt
gs://amp-pd-transcriptomics/inputs/reference/picard/gencode.v29.primary_assembly.annotation.RibosomalLocations.txt

File: rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  Int num_cpu_cores
+  String runtime_zones
+  Int preemptible_tries
+  
+  call collect_rna_seq_metrics {
+    input:
+      sample_name=sample_name,
+      bam_file=bam_file,
+      ref_flat_file=ref_flat_file,
+  	  ribosomal_intervals_file=ribosomal_intervals_file,
```
Comment (mbookman) [inline rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl]: There's an extra couple of spaces at the start of this line

File: rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  Int vm_disk_size_gb
+  String vm_memory
+  Int num_cpu_cores
+  String runtime_zones
+  Int preemptible_tries
+  
+  call collect_rna_seq_metrics {
+    input:
+      sample_name=sample_name,
+      bam_file=bam_file,
```
Comment (mbookman) [inline rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl:45]: Add a blank line after this one to separate the per-sample inputs from the reference inputs.

File: rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+##     falling back to a full-price VM.
+##
+## Outputs:
+## - Per-sample:
+##   - <sample-id>.RNA_Metrics
+
+workflow RNACollectRnaSeqMetrics {
+
+  String sample_name
+  File bam_file
```
Comment (mbookman) [inline rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl:29]: Add a blank line after this one to separate the per-sample inputs from the reference inputs.

File: rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+  output {
+    File metrics_output_path = collect_rna_seq_metrics.metrics_output_path
+  }
+}
+
+task collect_rna_seq_metrics {
+
+  String sample_name
+  File bam_file
```
Comment (mbookman) [inline rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl:67]: Add a blank line after this one to separate the per-sample inputs from the reference inputs.

File: rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+    memory: vm_memory
+    cpu: num_cpu_cores
+    preemptible: preemptible_tries
+    zones: runtime_zones
+  }
+  output {
+    File metrics_output_path = "${sample_name}.RNA_Metrics"
+  }
+}
```
Comment (mbookman) [inline rna/rna-collect-rna-seq-metrics/rna-collect-rna-seq-metrics.wdl]: nit: add a newline at the end of the file

**Review by mbookman — CHANGES_REQUESTED** [review]

**Review by mbookman — APPROVED** [review]
LGTM

**Review by ha-duong-technome — APPROVED** [review]

**Pull request merged**